bundle.md

```diff
@@ -12,19 +12,19 @@ A standard container bundle is made of the following 3 parts:
 
 # Directory layout
 
-A Standard Container bundle is a directory containing all the content needed to load and run a container. This includes its configuration file (`config.json`) and content directories. The main property of this directory layout is that it can be moved as a unit to another machine and run the same container.
+A Standard Container bundle is a directory containing all the content needed to load and run a container.
+This includes two configuration files `config.json` and `runtime.json`, and a rootfs directory.
+The `config.json` file contains settings that are host independent and application specific such as security permissions, environment variables and arguments.
+The `runtime.json` file contains settings that are host specific such as memory limits, local device access and mount points.
+The goal is that the bundle can be moved as a unit to another machine and run the same container if runtime.json is removed or reconfigured.
```

Comment:
On Tue, Aug 04, 2015 at 09:55:12PM -0700, Brandon Philips wrote:
> In the appc spec the thing that is contained and executed is called "the app".

So you have:

* bundles, which get written ahead of time, distributed, and used to
  seed the runtime
* containers, which are the environment setup for the application
  (namespaces, cgroups, mounts, …)
* applications, which are the process trees launched inside the
  containers

That seems reasonable to me.  Should I PR a glossary?

> "The goal is that the bundle can be moved, as a unit, to another
> machine and the app is executed in a similar manner on the new host
> if `runtime.json` is removed or reconfigured."

That's easier for me to read, but “in a similar manner” is wishy
enough that I'm not sure what the sentence clarifies.  But that may
just be because I don't see a clear line around “runtime-specific”
[1].

[1]: https://github.com/opencontainers/specs/pull/88#issuecomment-126516625